tests/tasks/test_api_create_task.py

```diff
@@ -4,9 +4,47 @@
 from app import messages
 from tests.tasks.tasks_base_setup import TasksBaseTestCase
 from tests.test_utils import get_test_request_header
-
+from datetime import timedelta
+from time import sleep
 
 class TestCreateTaskApi(TasksBaseTestCase):
+    # User not involved in mentorship relation tries to create a task (FAIL)
+    # gives 400, UNACCEPTED_STATE_RELATION response
+    def test_create_task_api_not_in_relation(self):
+        auth_header = get_test_request_header(self.first_user.id)
+        expected_response = messages.UNACCEPTED_STATE_RELATION
+        actual_response = self.client.post('/mentorship_relation/%s/task' % self.mentorship_relation_without_first_user.id,
+                                           follow_redirects=True, headers=auth_header, content_type='application/json',
+                                           data=json.dumps(dict(description=self.test_description)))
+        self.assertEqual(400, actual_response.status_code)
+        self.assertDictEqual(expected_response, json.loads(actual_response.data))
+
+    # Valid user tries to create task without description in the body (FAIL)
+    # gives 400, DESCRIPTION_FIELD_IS_MISSING response
+    def test_create_task_api_no_description(self):
+        auth_header = get_test_request_header(self.first_user.id)
+        expected_response = messages.DESCRIPTION_FIELD_IS_MISSING
+        actual_response = self.client.post('/mentorship_relation/%s/task' % self.mentorship_relation_without_first_user.id,
```

Comment:
Here  `self.mentorship_relation_without_first_user` can be replaced with `self.mentorship_relation_w_second_user` since we just want to check for the description field.